actors/builtin/miner/miner_actor.go

```diff
@@ -1654,12 +1662,14 @@ func handleProvingDeadline(rt Runtime) {
 
 	var st State
 	rt.State().Transaction(&st, func() interface{} {
+
 		var err error
 		{
 			// Vest locked funds.
 			// This happens first so that any subsequent penalties are taken from locked pledge, rather than free funds.
 			newlyVested, err = st.UnlockVestedFunds(store, rt.CurrEpoch())
```

Comment:
Remove `newlyVested` from the top-level scope so it can't be accidentally processed again.

The comment above this line is now misleading.